Rovers/Rovers.tex

```diff
@@ -0,0 +1,249 @@
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project would involve divvying out the different project areas to the different team members or small groups and each week meeting to discuss progress. 
+The team would take this information, refine it and put it into the technical report along with construction.  
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The primary deliverables of this project will be the technical report and the rover itself by the end of the second semester. 
+It is also noted that the materials that the team will come across or notes should also be saved in an Google Drive folder for future reference.  
+The team will also present at SPEX design reviews and the weekly checkups at general meetings with the areas the team members are working on that week.
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+The largest milestones for the technical report would be the would be draft and completion. 
+The rover itself would include the various CAD models of the rover, software milestones, electrical system completion, and drive train system. 
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+The project is going to require members that are knowledgeable in mechanical design and manufacturing, software engineering, machine learning, electrical engineering, and project management. These members will also need to be experienced in research and design document work. Though this seems simple, finding SPEX members to do this well may be a challenge duye to competition from other projects. 
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+
+The rover project is looking at a minimal funding amount of \$500.00 USD from RIT SPEX. In order to give a better deliverable the team will also be reaching out to a large list of companies looking for further funding through sponsorship. 
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Support from faculty could greatly advance this study and what RIT SPEX is capable robotics wise. The team should reach out to professors for advice and help.
+
+\subsection{Long-Term Vision}
```

Comment:
Lots of bits on this from earlier in the document. Consider talking about URC and how this project can prepare SPEX for building a competition-grade rover in the future.